src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingBarracks.java

```diff
@@ -144,6 +131,68 @@ public void onUpgradeComplete(final int newLevel)
         super.onUpgradeComplete(newLevel);
     }
 
+    /**
+     * Return list of all the Barrack's Towers.
+     *
+     * @return a tuple with position and facing.
+     */
+    private List<Tuple<BlockPos, EnumFacing>> getBarracksTowers()
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "getBarracksTowers" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)